openmetadata-ui/src/main/resources/ui/src/styles/components/step.less

```diff
@@ -36,63 +32,62 @@
 }
 
 .ingestion-rounder.completed::after {
+  font-size: 12px;
   content: '\2713';
   display: block;
   margin-top: -4px;
   font-weight: 900;
   color: #7147e8;
 }
 
-.ingestion-line {
-  top: 10px;
-  bottom: 0;
-  position: absolute;
-  width: 75%;
-  height: 2px;
-  left: 82px;
-  background-color: #d9ceee;
-}
-.ingestion-line-se {
+.ingestion-deploy-rounder {
   display: block;
-  width: 2px;
-  height: 15px;
-  background-color: #d9ceee;
-  transform: translate(8px, 0);
+  width: 56px;
+  height: 56px;
+  border-radius: 50%;
+  margin-top: 0.15rem;
+  z-index: 1;
 }
 
-.service-stepper-line {
-  width: 80%;
-  left: 70px;
+.ant-steps-label-vertical .ant-steps-item-tail {
+  padding: 3.5px 0px;
 }
 
-.edit-ingestion-line {
-  width: 65%;
-  left: 100px;
+.ant-steps-item-finish
+  > .ant-steps-item-container
+  > .ant-steps-item-tail::after {
+  background-color: #d9ceee;
+  height: 2px;
 }
 
-.add-service-line {
-  width: 67%;
-  left: 108px;
+.ant-steps-item-process
+  > .ant-steps-item-container
+  > .ant-steps-item-tail::after {
+  background-color: #d9ceee;
+  height: 2px;
 }
 
-.add-ingestion-line {
-  width: 68%;
-  left: 90px;
+.ant-steps-small .ant-steps-item-title {
+  font-size: 12px;
+  color: #7147e8;
```

Comment:
Can we use store these colours to the less variable and use them here?

Reply:
Done